gradle/docs/_data/projects.yml

```diff
@@ -0,0 +1,23 @@
+- name: "AB-1"
+  url: https://se-edu.github.io/addressbook-level1
+
+- name: "AB-2"
+  url: https://se-edu.github.io/addressbook-level2
+
+- name: "AB-3"
+  url: https://se-edu.github.io/addressbook-level3
+
+- name: "AB-4"
+  url: "https://se-edu.github.io/addressbook-level4"
+
+- name: "Duke"
+  url:  https://se-edu.github.io/duke
+
+- name: "Collate"
+  url: https://se-edu.github.io/collate
+
+- name: "Book"
+  url: https://se-edu.github.io/se-book
+
+- name: "Resources"
+  url: https://se-edu.github.io/resources
```

Comment:
Let's remove the se-edu nav bar from duke. It's a starter template, not part of the se-edu website.

Reply:
Removed the nav bar for now. Technically we could toggle it in the config files but it's better to have less.